Bio/Wise/__init__.py

```diff
@@ -105,7 +106,7 @@ def align(
     if debug:
         sys.stderr.write("%s\n" % cmdline_str)
 
-    status = os.system(cmdline_str) >> 8
+    status = subprocess.call(cmdline_str, shell=True) >> 8
```

Comment:
Does subprocess definitely return the same value as os.system? There are weird and wonderful corner cases like minus values for system errors etc, and I am unclear what the ``>> 8`` was for.

Reply:
I'm not sure. `>> 8` is checking for error codes `>=256?`

Reply:
CC @mdehoon - would a return code outside 0-255 indicate it was the OS not the tool?

Reply:
if someone can clarify the desired behaviour, perhaps the logic can be written and commented to improve readability, otherwise I suggest maybe we just don't alter this?

Reply:
OK, leave as it is pending clarification of intent.